### ISSUE TITLE
Need to upgrade ember-css-modules to 0.3.3 to remove deprecation notice

### ISSUE THREAD

**Issue by rox163** [mansona/ember-cli-notifications#96]
Release 0.3.3 https://github.com/salsify/ember-css-modules/commits/v0.3.3 has the necessary upgrade to get rid of the init() deprecation notice. 

```
DEPRECATION: Overriding init without calling this._super is deprecated. Please call `this._super.init && this._super.init.apply(this, arguments);` addon: `ember-css-modules`
    at Function.Addon.lookup (/Users/rpanthak/git_workspace/ember-frost-modal/node_modules/ember-cli/lib/models/addon.js:896:27)

```

**Comment by ynnoj** [comment]
Hey, sorry about the troubles! I just released `4.0.6` which should fix this issue.

Let me know how it works out for you.

**Comment by rox163** [comment]
Works great! Thanks fro the quick update!
